SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -713,6 +713,20 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 			AActor* Actor = PriorityActors[j]->ActorInfo->Actor;
 			bool bIsRelevant = false;
 
+			// If the actor channel can't be found in the current connection (e.g. if the actor was detached from the controller),
+			// then search through all the connections to find the actor's channel.
```

Comment:
This is also a workaround. Perhaps we should be storing all actor channels in a single source regardless of connection. Could you please add another Jira task to tackle this.

Reply:
Done so 👍